modules/test/playwright/tests/style-book-web/previewStyleBookOnDisplayPageTemplates.spec.ts

```diff
@@ -0,0 +1,324 @@
+	displayPageTemplatesPage: DisplayPageTemplatesPage,
+	page: Page,
+	pageEditorPage: PageEditorPage,
+	styleBooksPage: StyleBooksPage,
+	templateName: string
+) => {
+	const publishButton = page.getByRole('button', {
+		name: 'Publish',
+	});
+
+	await styleBooksPage.goto();
+	await page.getByRole('button', {name: /more actions/i}).click();
+	await page.getByRole('menuitem', {name: 'Edit'}).click();
+	await page.getByRole('button', {name: 'Color System'}).click();
+	await page.getByRole('menuitem', {name: 'General'}).click();
+
+	const colorInput = page
+		.getByLabel('Body Color')
+		.getByLabel('Color')
+		.locator('.layout__color-picker__input');
+	await fillAndClickOutside(page, colorInput, colorHEX);
+	expect(page.getByText('Saved')).toBeVisible();
+
+	// Publish and verify the style
+
+	const publishTemplate = async () => {
+		await publishButton.waitFor();
+		await publishButton.click();
+		await page
+			.getByRole('dialog')
+			.getByRole('button', {name: /publish/i})
+			.click();
+		await waitForSuccessAlert(page, 'Success:Your request');
+	};
+	await publishTemplate();
+
+	await displayPageTemplatesPage.goto();
+
+	await page.getByRole('link', {name: templateName}).click();
+	await page.getByLabel('Page Design Options').click();
+	await page.getByRole('tab', {name: 'Style Book'}).click();
+	await page.getByLabel('Test Style Book Name').click();
+
+	const headingId = await pageEditorPage.getFragmentId('Heading');
+
+	const fragmentColor = await pageEditorPage.getFragmentStyle({
+		fragmentId: headingId,
+		isTopperStyle: true,
+		style: 'color',
+	});
+	expect(fragmentColor).toBe(colorRGB);
+};
+
+test('Create Style Book and View the Fragments is selected in the preview type selector', async ({
+	page,
+	styleBooksPage,
+}) => {
+	const publishButton = page.getByRole('button', {
+		name: 'Publish',
+	});
+	await styleBooksPage.goto();
+	await styleBooksPage.createStyleBook('Test Style Book Name');
+	await publishButton.click();
+	await page
+		.getByRole('dialog')
+		.getByRole('button', {name: /publish/i})
+		.click();
+
+	await page.getByRole('button', {name: /more actions/i}).click();
+	await page.getByRole('menuitem', {name: 'Edit'}).click();
+	await expect(page.getByRole('button', {name: 'Pages'})).toBeVisible();
+});
+
+test('Add a Heading fragment to the first display page template for Web Content Article and Basic Web Content and view the Display Page Templates is shown in preview type selector', async ({
+	displayPageTemplatesPage,
+	page,
+	pageEditorPage,
+	styleBooksPage,
+}) => {
+	const publishButton = page.getByRole('button', {name: 'Publish'});
+
+	// Create and publish a new page template
+
+	await displayPageTemplatesPage.goto();
+	await displayPageTemplatesPage.createTemplate({
+		contentSubtype: 'Basic Web Content',
+		contentType: 'Web Content Article',
+		name: 'Web Content DPT 1',
+	});
+
+	const pageTemplate = page.getByRole('link', {
+		name: 'Web Content DPT 1',
+	});
+
+	await pageTemplate.click();
```

Comment:
There is already a method for editing the template (displayPageTemplatesPage.editTemplate). It's probably better to use that.